tests/relay/test_schema.py

```diff
@@ -25,6 +25,21 @@ def test_schema():
     assert schema_output == expected
 
 
+def test_schema_future_annotations():
+    from .schema_future_annotations import schema
+
+    schema_output = str(schema).strip("\n").strip(" ")
+    output = pathlib.Path(__file__).parent / "schema.gql"
```

Comment:
I think it is probably better to export it to a different path than the base one, or else any change done there will also need to be done in the future annotations file.

```suggestion
    output = pathlib.Path(__file__).parent / "future_annotations_schema.gql"
```

Reply:
the idea is to plainly copy schema.py and add `from __future__ import annotations` to it.
So it should match schema.gql

Reply:
it may should be done more sophisticated